source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
+		if (x < 0 || x >= this width)
+			raise("Column index out of range in FloatMatrix setColumn")
+		if (values count != this height)
+			raise("Number of values must match matrix height in FloatMatrix setColumn")
+		y := 0
+		values each (|value|
```

Comment:
This is a closure, which allocates memory which will never be freed, and so it will leak memory.